.github/dependabot.yml

```
@@ -0,0 +1,18 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/" # Location of package manifests
```

Unsure what package manifests mean here, might just delete the comment?
```suggestion
    directory: "/"  # Location of package manifests
```

---

Removed in 34f72aa9d4732f72d385b01e2d00df8e670ce968